index.js

```diff
@@ -1,30 +1,43 @@
 var fs = require('fs'),
     path = require('path'),
-    existsSync = require('fs').existsSync || require('path').existsSync;
+    semver = require('semver'),
+    existsSync = require('fs').existsSync || path.existsSync;
 
-var versions = [
- '2.0.0',
- '2.0.1',
- '2.0.2',
- '2.1.0',
- '2.1.1',
- '2.2.0',
- '2.3.0',
- '3.0.0',
- '3.0.3',
- '3.0.4'
-];
 
-module.exports.versions = versions;
+var getVersions = function () {
+    var names = fs.readdirSync('./'), versions = [];
+    for (var i = 0; i < names.length; i++) {
+        if(names[i].match(/^\d{1,2}\.\d{1,2}\.\d{1,2}$/)) versions.push(names[i]);
```

Comment:
is this trying to figure out if the string is a valid semver? should this use `semver.valid` in a call to `.filter()`